morebits.js

```diff
@@ -4595,7 +4596,7 @@ Morebits.wikitext.page.prototype = {
 			regex = regex.join('|');
 		}
 
-		flags = flags || 'i';
+		flags = flags || flags === '' ? '' : 'i';
```

Comment:
Probably cleaer to write as `if (typeof flags !== 'string') flags = 'i';`

Reply:
Much!  Done.